optashift-employee-rostering-gwtui/src/main/java/org/optaplanner/openshift/employeerostering/gwtui/client/pages/spotroster/ShiftBlobPopoverContent.java

```diff
@@ -122,35 +121,35 @@ public void init(final BlobView<?, ?> blobView) {
 
         this.blobView = (ShiftBlobView) blobView;
         final ShiftBlob blob = (ShiftBlob) blobView.getBlob();
-        final Shift shift = blob.getShift();
+        final ShiftView shift = blob.getShiftView();
 
         employeeSelect.clear();
         employeeSelect.addItem("Unassigned", "-1"); //FIXME: i18n
 
         // TODO: Do rest call
         SpotRestServiceBuilder.getSpotList(tenantStore.getCurrentTenantId(), FailureShownRestCallback.onSuccess(spots -> {
             spots.forEach(s -> this.spotSelect.addItem(s.getName(), s.getId().toString()));
-            spotSelect.setSelectedIndex(spots.indexOf(shift.getSpot()));
+            spotSelect.setSelectedIndex(spots.indexOf(blob.getSpot()));
         }));
 
         EmployeeRestServiceBuilder.getEmployeeList(tenantStore.getCurrentTenantId(), FailureShownRestCallback.onSuccess(employees -> {
             this.employeesById = employees.stream().collect(Collectors.toMap(Employee::getId, Function.identity()));
             employees.forEach(e -> employeeSelect.addItem(e.getName(), e.getId().toString()));
-            employeeSelect.setSelectedIndex((shift.getEmployee() == null) ? 0 : employees.indexOf(shift.getEmployee()) + 1);
+            employeeSelect.setSelectedIndex((blob.getEmployee() == null) ? 0 : employees.indexOf(blob.getEmployee()) + 1);
         }));
 
-        final OffsetDateTime start = shift.getStartDateTime();
+        final LocalDateTime start = shift.getStartDateTime();
         fromDay.value = start.getMonth().toString() + " " + start.getDayOfMonth(); //FIXME: i18n
-        fromHour.value = GwtJavaTimeWorkaroundUtil.toLocalTime(start) + "";
+        fromHour.value = start + "";
```

Comment:
don't concatonate "" just to cast to a String. Use Integer.toString(start) instead.